auxx/src/Command/Types.hs

```diff
@@ -38,7 +39,7 @@ data SendToAllGenesisParams = SendToAllGenesisParams
 
 -- | Parameters for 'ProposeUpdate' command.
 data ProposeUpdateParams = ProposeUpdateParams
-    { puIdx             :: Int -- TODO: what is this? rename
+    { puSecretKeyIdx    :: Int -- the node that creates/signs the proposal
```

Comment:
I wouldn't call it _the node_, it's the index of one of the secrets from our keyfile (`UserSecret`).